### PR TITLE
Add a way to get and set gamemode of the player

### PR DESCRIPTION
**Pull request by Xephi**
Player api do not have currently a way to get (or set) the gamemode of the player :/

If someone can add implementation of that ^^"

### CODE REVIEW

**Comment by ST-DDT** [comment]
Also included there, https://github.com/SpongePowered/SpongeAPI/pull/368.
But i like the single purpose PRs.

File: src/main/java/org/spongepowered/api/entity/player/Player.java [hunk 1]
```diff
@@ -117,5 +118,19 @@
      * Removes the currently displayed {@link Title} from the player's screen.
      */
     void clearTitle();
+    
+    /**
+     * Set the player {@link GameMode}.
+     * 
+     * @param gamemode The {@link GameMode} to set
+     */
+     void setGameMode(GameMode gm);
```
Comment (simon816) [inline src/main/java/org/spongepowered/api/entity/player/Player.java]: Parameter name does not match up with javadoc `@param`
I suggest changing it to `gameMode`

File: src/main/java/org/spongepowered/api/entity/player/Player.java [hunk 2]
```diff
@@ -117,5 +118,19 @@
...
+
+    /**
+     * Get the player {@link GameMode}.
```
Comment (Phroa) [inline src/main/java/org/spongepowered/api/entity/player/Player.java]: you may want to change `player` to `player's current`

File: src/main/java/org/spongepowered/api/entity/player/Player.java [hunk 3]
```diff
@@ -117,5 +118,19 @@
      * Removes the currently displayed {@link Title} from the player's screen.
      */
     void clearTitle();
+    
+    /**
+     * Set the player {@link GameMode}.
```
Comment (Phroa) [inline src/main/java/org/spongepowered/api/entity/player/Player.java]: `player` -> `player's`?

**Comment by Zidane** [comment]
Already added in #368 

Thanks anyways!

**Pull request closed without merging**